src/core/qgismobileapp.cpp

```diff
@@ -149,6 +149,8 @@
 #include <qgsproject.h>
 #include <qgsprojectstorage.h>
 #include <qgsprojectstorageregistry.h>
+
+#include <ZXing/ReadBarcode.h>
```

Comment:
Is this include used for anything?

Reply:
Oups, it was used to test things here, will remove.